relevanceai/dataset/vis/plot.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Optional, Union
+
+import numpy as np
+import pandas as pd
+
+import plotly.express as px
+
+from relevanceai.dataset.statistics.statistics import Statistics
+
+
+class Plot(Statistics):
+    def plot(
```

Comment:
What plot is it? Can we be specific here because I think we are going to add a lot of plots in the future